examples/create-react-app/package.json

```diff
@@ -39,5 +39,10 @@
   },
   "devDependencies": {
     "tailwindcss": "^3.3.2"
+  },
+  "resolutions": {
+    "minimist": "^1.2.6",
+    "nth-check": "^2.0.1",
+    "semver": "^7.5.2"
```

Comment:
Why not just make these dependencies rather than resolutions?

Reply:
If I understand it right, it is because it is sub-dependencies (dependencies of other packages dependencies). We don't have these packages in our projects.
So if we add these packages to our dependencies and specify secure versions, it would not prevent other dependencies from installing and using insecure versions of these packages.